WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+Extensions.swift

```diff
@@ -288,8 +288,12 @@ extension NotificationsViewController: WPTableViewHandlerDelegate
         // Update Separators:
         // Due to an UIKit bug, we need to draw our own separators (Issue #2845). Let's update the separator status
         // after a DB OP. This loop has been measured in the order of milliseconds (iPad Mini)
+
         for indexPath in tableView.indexPathsForVisibleRows ?? [] {
-            let cell = tableView.cellForRowAtIndexPath(indexPath) as! NoteTableViewCell
+            guard let cell = tableView.cellForRowAtIndexPath(indexPath) as? NoteTableViewCell else {
+                continue
```

Comment:
Just my 2 cents but:

If the cells are really supposed to be `NoteTableViewCell`, then I'd at least add an assertion for debug builds before `continue`.  Otherwise we'd just be hiding an error.